multiples_sum.js

```diff
@@ -0,0 +1,23 @@
+function multSum(valOne, valTwo, max) {
+  var sumOne = 0;
+  var sumTwo = 0;
+  var numOne = 1;
+  var numTwo = 1;
+  // finding all multiples of valOne
+  do {
+    var prodOne = valOne * numOne;
+    numOne++;
+    sumOne += prodOne;
+  } while (prodOne < max);
+  // finding all multiples of valTwo
+  do {
+    var prodTwo = valTwo * numTwo;
+    numTwo++;
+    sumTwo += prodTwo;
+  } while (prodTwo < max);
+  var sum = sumOne + sumTwo;
+  return sum;
+}
+
+console.log(multSum(3, 5, 1000));
+// answer: 268335
```

Comment:
Did this answer work when you submitted it to Project Euler? My answer is different from this.

Reply:
You can submit answers? I'm still learning to navigate the site, I thought it was just an archive of logic problems

Reply:
It is a _very_ old site with poor UX lol. Yeah you can create an account and submit your answer for verification.